### ISSUE TITLE
Audit #1 – Share a single file  – Custom select "Rolle" not fully keyboard accessible

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4407]
Connected to Audit 1, see #4300 
1.5 Download a single file, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
The "Rolle:" widget inside the "add people" dialog (see screenshot) is not fully keyboard accessible. While it looks like a select/custom select/combobox it is actually something of a disclosure widget, and the DOM structure fits. You can open/trigger it with the space key, but the items inside the newly visible container ("Betrachter", "Editor", "Zusätzliche Rechte" are not reachable by keyboard.
![Screenshot 2020-12-02 at 08 13 06](https://user-images.githubusercontent.com/505181/100840474-38489c00-3476-11eb-8c74-b961b2fb061e.png)

# Remediation
Currently, `.files-collaborators-roles-dropdown` uses `<li>`s with a click event. Change this to `<button>`s instead. Alternatively, see #4790

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
